x-pack/plugins/security_solution/public/detections/components/rules/step_define_rule/index.tsx

```diff
@@ -223,6 +214,30 @@ const StepDefineRuleComponent: FC<StepDefineRuleProps> = ({
   const machineLearningJobId = formMachineLearningJobId ?? initialState.machineLearningJobId;
   const queryBar = formQuery ?? initialState.queryBar;
 
+  const setRuleTypeCallback = useSetFieldValueWithCallback({
+    field: 'ruleType',
+    value: ruleType,
+    setFieldValue,
+  });
+
+  const handleSetRuleFromTimeline = useCallback(
+    ({ index: timelineIndex, queryBar: timelineQueryBar }) => {
+      const setQuery = () => {
+        setFieldValue('index', timelineIndex);
+        setFieldValue('queryBar', timelineQueryBar);
```

Comment:
Should we be copying over the EQL settings as well in effort to best match the Timeline results?
Timeline:
<p align="center">
  <img width="500" src="https://user-images.githubusercontent.com/2946766/219491034-fdea4ed1-1d12-4332-90ad-d2922b590b95.png" />
</p>
Rule:
<p align="center">
  <img width="500" src="https://user-images.githubusercontent.com/2946766/219490857-e300c5fa-83ab-4c0d-ae66-2e3ab0b68f4a.png" />
</p>

Reply:
Oh cool, yeah I can do that. will let you know when ive updated the branch!